Frontend/ui-common/src/Engineering/Constants.purs

```diff
@@ -48,4 +48,7 @@ defaultSeparatorCount :: Int
 defaultSeparatorCount = 4
 
 globalPayload :: String
-globalPayload = "__payload"
\ No newline at end of file
+globalPayload = "__payload"
+
+whatsAppSupportLink :: String
```

Comment:
where are we showing whatsapp support here?

Reply:
I am using this in the help and support screen for the whatsapp support for the driver